src/main.ml

```diff
@@ -127,6 +127,13 @@ let synjitsu_domain_uuid =
      MirageOS TCP/IP stack."  in
   Arg.(value & opt (some string) None & info ["synjitsu"] ~docv:"UUID" ~doc)
 
+let persistdb =
+  let doc =
+    "Store the Irmin database in the specified path. The default is to store the database in memory only. \
+     Note that modifying this database while Jitsu is running is currently unsupported and may crash Jitsu
+     or have other unexpected results." in
+  Arg.(value & opt (some string) None & info [ "persistdb" ] ~docv:"path" ~doc)
+
```

Comment:
It would be very cool if Irmin had a `irmin.cmdliner` sub-package that gave a standard set of command line options for common things like persistence, the database path and so on. /cc @samoht